webpack/webpack-embedded-dependencies-plugin/README.md

```diff
@@ -0,0 +1,142 @@
+# webpack-embedded-dependencies-plugin
+
+## Installation
+
+`npm install @rushstack/webpack-embedded-dependencies-plugin --save`
+
+## Overview
+A webpack plugin for generating a list of embedded dependencies. Embedded dependencies are third-party packages which are being
+bundled into your released code and are often times subject to license, security, and other legal requirements. This plugin
+aims to make it easier to generate a list of embedded dependencies and their associated metadata, so they can be analyzed by additional tools.
+
+The plugin also includes the ability to generate a secondary asset which contains the license text for each embedded dependency into a single file called
+THIRD-PARTY-NOTICES.html. This is a common legal requirement for large companies deploying commercial services/products containing open source code.
```

Comment:
HTML? Seems like the name and output format should be configurable.

Reply:
Oh, the name is configurable. This summary seems to imply that it isn't.

HTML seems like a weird default choice.

Reply:
When reviewing existing implementations, each major product was generating an HTML file. Since this is an optional generation, having the most common scenario covered felt right to me.